seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -893,68 +893,119 @@
+        "  this.head = null; ",
         "",
         "  var Node = function(element){",
         "    this.element = element; ",
         "    this.next = null; ",
         "  }; ",
         "",
-        "  this.head = function(){",
-        "    return head;",
-        "  };",
-        "",
-        "  this.size = function(){",
-        "    return length;",
-        "  };",
-        "",
         "  this.add = function(element){",
         "    // Only change code below this line",
         "",
         "    // Only change code above this line",
-        "  };",
-        "};"
+        "  }; ",
+        "} ",
+        "",
+        "// Test your code",
+        "var conga = new LinkedList();",
+        "conga.add('Kitten');",
+        "console.log(conga.head.element);",
+        "conga.add('Puppy');",
+        "console.log(conga.head.next.element);"
       ],
       "tests": [
-        "assert((function(){var test = new LinkedList(); return (typeof test.add === 'function')}()), 'message: Your <code>LinkedList</code> class should have a <code>add</code> method.');",
-        "assert((function(){var test = new LinkedList(); test.add('cat'); return test.head().element === 'cat'}()), 'message: Your <code>LinkedList</code> class should assign <code>head</code> to the first node added.');",
-        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); return test.head().next.element === 'dog'}()), 'message: The previous <code>node</code> in your <code>LinkedList</code> class should have reference to the newest node created.');",
-        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); return test.size() === 2}()), 'message: The  <code>size</code> of your <code>LinkedList</code> class should equal the amount of nodes in the linked list.');"
+        "assert((function(){var test = new LinkedList(); return (typeof test.add === 'function')}()), 'message: Your <code>LinkedList</code> class should have an <code>add</code> method.');",
+        "assert((function(){var test = new LinkedList(); test.add('cat'); return test.head.element === 'cat'}()), 'message: Your <code>LinkedList</code> class should assign the first node you add to the <code>head</code> of your list.');",
+        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); return test.head.next.element === 'dog'}()), 'message: The previous <code>node</code>'s <code>next</code> property in your <code>LinkedList</code> class should have reference to the newest node you created.');",
+        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); return test.length === 2}()), 'message: The  <code>length</code> of your <code>LinkedList</code> class should equal the number of nodes in the linked list.');"
       ],
       "solutions": [],
       "hints": [],
       "type": "waypoint",
       "challengeType": 1,
-      "translations": {}
+      "translations": {
```

Comment:
Please revert this line of change. We do not need to add empty translation blocks.